src/public/parts/shells/social/public/src/templates/settings_muted_keywords.html

```
@@ -1,13 +1,15 @@
-			<input type="checkbox" id="mutedKeywordMatchTags" checked />
-			<span data-i18n="social.settings.mutedKeywordsMatchTags"></span>
-		</label>
-		<button type="submit" class="btn btn-primary btn-sm" data-i18n="social.settings.mutedKeywordsAdd"></button>
-	</form>
-	<div class="muted-keyword-chips" id="mutedKeywordChips"></div>
+<section class="settings-card card bg-base-200 shadow-sm">
+	<div class="card-body gap-2">
+		<h3 class="card-title text-base" data-i18n="social.settings.mutedKeywordsTitle"></h3>
+		<p class="settings-hint text-sm opacity-70" data-i18n="social.settings.mutedKeywordsHint"></p>
+		<form id="mutedKeywordForm" class="join join-vertical sm:join-horizontal w-full gap-2 flex flex-wrap items-center">
+			<input type="text" id="mutedKeywordInput" class="input input-bordered input-sm join-item flex-1 min-w-40" maxlength="64" data-i18n="social.settings.mutedKeywords" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)